server/data/incidentClient.js

```diff
@@ -46,6 +46,17 @@ const updateDraftReport = (reportId, incidentDate, formResponse) => {
 const maxSequenceForBooking =
   '(select max(r2.sequence_no) from report r2 where r2.booking_id = r.booking_id and user_id = r.user_id)'
 
+const updateAgencyId = (agencyId, username, bookingId) => {
+  return nonTransactionalClient.query({
+    text: `update v_report r
+                set agency_id = COALESCE($1,   r.agency_id)
```

Comment:
obviously no actual need for the COALESCE as validation wouldn't allow ever setting it to null but no harm

Reply:
ok